.github/workflows/main.yml

```diff
@@ -51,16 +51,27 @@ jobs:
     - name: dotnet build TestProject
       run: dotnet build ./test/TestProject/TestProject.csproj -c Release
     
-    # Package
-    - name: dotnet pack
+    # Package SDK
+    - name: dotnet pack SDK
       run: dotnet pack -c Release src/MSBuild.Sdk.SqlProj/MSBuild.Sdk.SqlProj.csproj
 
-    # Upload package
-    - name: upload
+    # Upload SDK package
+    - name: upload SDK
       uses: actions/upload-artifact@v1
       with:
         name: nuget-package
         path: ./src/MSBuild.Sdk.SqlProj/bin/Release/
+    
+    # Package TestProject
+    - name: dotnet pack TestProject
+      run: dotnet pack -c Release ./test/TestProject/TestProject.csproj
+
+    # Upload TestProject package
+    - name: upload TestProject
+      uses: actions/upload-artifact@v1
+      with:
+        name: nuget-package
```

Comment:
Nit: We might want to rename this artifact to nuget-packages since it now contains multiple packages.

Reply:
Maybe I'm misunderstanding, but I don't believe it does. `TestProjectWithSDKRef` now contains multiple packages, but `TestProject` is still just the one: `TestProject/bin/release/netstandard2.0/TestProject.version.nupkg`.

Reply:
You're right, but the MSBuild.Sdk.SqlProj package is also uploaded using the same name a little bit further up in the workflow file. I guess GitHub actions combines both packages into the same artifact (somehow), which is fine, but we should then at least make it plural to indicate what it is ;).

Reply:
Oh, I see. Okay, no problem.